src/main/java/org/codehaus/plexus/components/io/attributes/PlexusIoResourceAttributeUtils.java

```diff
@@ -183,141 +174,11 @@ public static PlexusIoResourceAttributes getFileAttributes( File file )
         return o;
     }
 
-    public static Map<String, PlexusIoResourceAttributes> getFileAttributesByPath( File dir )
```

Comment:
Actually I've deleted this one by mistake - it does make sense and it's used by Plexus Archiver. I'll restore it.